examples/.experimental/next-app-dir/src/app/client/ClientGreeting.tsx

```diff
@@ -0,0 +1,12 @@
+'use client';
+
+// import { use } from 'react';
+// import { api } from 'trpc-api';
+
+export function ClientGreeting() {
+  // const greeting = use(api.greeting.query({ text: 'from client' }));
+
+  // return <>{greeting}</>;
+
+  return <>not implemented</>;
+}
```

Comment:
Wrote about this here; https://github.com/vercel/next.js/issues/51031

But our previous workaround with storing the promise as a ref etc seems to not be supported anymore...

Leaving this for now and we'll take this back up when doing the RQ stuff?